SPECS/mock-core-configs/mock-core-configs.spec

```diff
@@ -1,54 +1,76 @@
-Summary:    Mock core config files basic chroots
+%if 0%{?el8}
+%global python3 /usr/libexec/platform-python
+%endif
+
 Name:       mock-core-configs
-Version:    36.4
+Version:    41.2
 Release:    1%{?dist}
-License:    GPLv2+
+Vendor:     Microsoft Corporation
+Distribution: Azure Linux
+Summary:    Mock core config files basic chroots
+
+License:    GPL-2.0-or-later
 URL:        https://github.com/rpm-software-management/mock/
 # Source is created by
 # git clone https://github.com/rpm-software-management/mock.git
```

Comment:
Similar ask regarding the source tarball: can we use the upstream directly or create a script?

Reply:
Ah, good catch. These comments came from the Fedora version of the spec; I had the same concern and was able to directly use the upstream source archive directly. I'll appropriately update the comments. (The URLs of the archives listed in cgmanifest is accurate.)